lib/absinthe/middleware.ex

```diff
@@ -231,6 +232,11 @@ defmodule Absinthe.Middleware do
   end
   ```
 
+  **It is important to note that we are matching for the `:query`, `:subscription` or
+  `:mutation` identifier types. We do this because the middleware function will be
+  called for each field in the schema. It is also important to provide a fallback so
+  that the default `Absinthe.Middleware.MapGet` is called.**
```

Comment:
I would say "so that the default `Absinthe.Middleware.MapGet` is _configured_" since it isn't actually called at this point.

Reply:
done